### DISCUSSION TITLE
does this template auto update its dependencies on pnpm i?

### DISCUSSION THREAD

**Discussion by ClemensDinkel** [cawa-93/vite-electron-builder#804] [Q&A] [answered]
I have built a small app using this template a while ago and now recently this app has been moved to a monorepo along with another project. When I work on the other project and use `pnpm i` it installs the dependecies for all projects in the monorepo, which is intentional. However everytime I do this, this also applies some changes to the pnpm-lock.yaml in the root folder of the monorepo. This seems to happen only to packages used by the app I used this template for. 
So back to the original question: Does this template somehow auto update its dependencies when you install packages? If yes where can I configure this?

**Comment by cawa-93** [comment, marked as answer]
This template does not contain any pnpm-specific configuration. I don't quite understand what your problem is. Automatic update of dependencies works thanks to renovate. Its configuration is here https://github.com/cawa-93/vite-electron-builder/blob/main/.github/renovate.json

**Reply by ClemensDinkel** [reply]
Thanks for your quick reply. It seems the reason for the lockfile updates aren't connected to the configurations of this template. Renovate would do such a thing, but the renovate.json was removed on our part, so it can't be responsible. Anyway.
Stay safe
